lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3089,8 +3089,7 @@ class Cleanup {
   void apply(SILBuilder &builder, SILLocation loc) {
     if (!func) return;
     assert(value);
-    LLVM_DEBUG(getADDebugStream() << "Running `Cleanup::apply` for "
-               << value << '\n');
+    LLVM_DEBUG(getADDebugStream() << "Running `Cleanup::apply` for " << value);
```

Comment:
Regarding debug log changes:
* Old:
```
[AD] Running `Cleanup::apply` for   %7 = alloc_stack $τ_0_0.CotangentVector        // users: %56, %41, %8

[AD] Cleaning up local buffer   %7 = alloc_stack $τ_0_0.CotangentVector        // users: %56, %41, %8

[AD] Running `Cleanup::apply` for   %4 = alloc_stack $τ_0_0.CotangentVector        // users: %57, %5

[AD] Cleaning up local buffer   %4 = alloc_stack $τ_0_0.CotangentVector        // users: %57, %5

[AD] Generated adjoint
```

* New:
```
[AD] Running `Cleanup::apply` for   %7 = alloc_stack $τ_0_0.CotangentVector        // users: %56, %41, %8
[AD] Cleaning up local buffer   %7 = alloc_stack $τ_0_0.CotangentVector        // users: %56, %41, %8
[AD] Running `Cleanup::apply` for   %4 = alloc_stack $τ_0_0.CotangentVector        // users: %57, %5
[AD] Cleaning up local buffer   %4 = alloc_stack $τ_0_0.CotangentVector        // users: %57, %5
[AD] Generated adjoint.
```

It seems the printed `SILValue` always has a newline at the end - verified via visual inspection based on existing tests.

Reply:
Not sure if that's the case if it's a SIL argument.